distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -169,6 +189,12 @@ type Connection
        the table already existed), inserts the contents of the provided
        in-memory table and returns a handle to the newly created table.
 
+       ! Temporary Tables
+
+         Note that temporary tables may not be visible in the table catalog, so
+         some features which rely on it like the `Table.query` shorthand mode
+         may not work correctly with temporary tables.
+
```

Comment:
we will need to copy this warning to Postgres_Connection and SQLite_Connection too.